spec/forms/decidim/participatory_documents/suggestion_form_spec.rb

```diff
@@ -34,17 +46,17 @@ module ParticipatoryDocuments
       end
 
       it "is invalid with a body that is too short" do
-        form.body = "Short body"
+        form.body = "body"
 
         expect(form).to be_invalid
-        expect(form.errors[:base]).to include(I18n.t("activemodel.errors.models.suggestion.attributes.too_short", min_length: SuggestionForm::MIN_BODY_LENGTH))
+        expect(form.errors[:body]).to include(I18n.t("activemodel.errors.models.suggestion.attributes.too_short", min_length: Decidim::ParticipatoryDocuments.config.min_suggestion_length))
       end
 
       it "is invalid with a body that is too long" do
-        form.body = "Long body" * 30
+        form.body = "Long body" * 300
 
         expect(form).to be_invalid
-        expect(form.errors[:base]).to include(I18n.t("activemodel.errors.models.suggestion.attributes.too_long", max_length: SuggestionForm::MAX_BODY_LENGTH))
+        expect(form.errors[:body]).to include(I18n.t("activemodel.errors.models.suggestion.attributes.too_long", max_length: Decidim::ParticipatoryDocuments.config.max_suggestion_length))
```

Comment:
```suggestion
        expect(form.errors[:body]).to include(I18n.t("activemodel.errors.models.suggestion.attributes.too_long", max_length: 500))
```
Same here.

In addition, now we could for two additional situations:
- [ ] When it is modified in the component (in the current code this will fail)
  To do this, you will need to add some context with the current_component in the form, there are many examples in the codebase.
- [ ] When it is modified in the defaults, you can do it like this:

```ruby
context "when default min size is another" do
  let(:body) { "short" }
  before do
    allow(Decidim::ParticipatoryDocuments).to receive(:max_suggestion_length).and_return(7)
  end

  it "is expected to be invalid" do
    expect(form).to be_invalid
    expect(form.errors[:body]).to include("too short 7 chars bla bla"))
  end
  context "when body is ok" do
    let(:body) { "enough body" }
    is_expected_to be_valid
  end
end